### ISSUE TITLE
Error with database 119

### ISSUE THREAD

**Issue by DFGitH** [GjjvdBurg/signal2html#65]
Hello,
I'm trying to use signal2html version 0.2.9 and i have a bad issue.

![2022-04-20 11_49_20-C__Windows_System32_cmd exe](https://user-images.githubusercontent.com/104067242/164201442-b17c91d5-a9cd-4e72-88e9-cd5196479fbe.png)

Have you worked on this version ?

Thanks for a solution ;)

**Comment by GjjvdBurg** [comment]
What's the error you're getting?

**Comment by DFGitH** [comment]
Hello,
Nothing more that the image posted in my first post: "...version is untested, please report error".
I have nothing after this and no data output.
Have you test this version?

**Comment by GjjvdBurg** [comment]
Yes, I have tested this version and it works on my end. Are you not getting any threads in the output directory?

**Comment by DFGitH** [comment]
Hello,
No, nothing in the output directory...maybe the backup is corrupt.
Thanks for your time.
